### ISSUE TITLE
FIX: Salt error during setup - [ERROR   ] Unable to connect pusher: Stream is closed

### ISSUE THREAD

**Issue by m0duspwnens** [Security-Onion-Solutions/securityonion#7203]
```
Syncing all salt modules.
[INFO    ] Loading fresh modules for state activity
[INFO    ] Fetching file from saltenv 'base', ** done ** 'top.sls'
[INFO    ] Fetching file from saltenv 'base', ** done ** 'salt/minion.defaults.yaml'
[INFO    ] Creating module dir '/var/cache/salt/minion/extmods/modules'
[INFO    ] Syncing modules for environment 'base'
[INFO    ] Loading cache from salt://_modules, for base
[INFO    ] Caching directory '_modules/' for environment 'base'
[INFO    ] Fetching file from saltenv 'base', ** done ** '_modules/healthcheck.py'
[INFO    ] Fetching file from saltenv 'base', ** done ** '_modules/needs_restarting.py'
[INFO    ] Fetching file from saltenv 'base', ** done ** '_modules/so.py'
[INFO    ] Fetching file from saltenv 'base', ** done ** '_modules/telegraf.py'
[INFO    ] Fetching file from saltenv 'base', ** done ** '_modules/zeekctl.py'
[INFO    ] Copying '/var/cache/salt/minion/files/base/_modules/healthcheck.py' to '/var/cache/salt/minion/extmods/modules/healthcheck.py'
[INFO    ] Copying '/var/cache/salt/minion/files/base/_modules/needs_restarting.py' to '/var/cache/salt/minion/extmods/modules/needs_restarting.py'
[INFO    ] Copying '/var/cache/salt/minion/files/base/_modules/so.py' to '/var/cache/salt/minion/extmods/modules/so.py'
[INFO    ] Copying '/var/cache/salt/minion/files/base/_modules/telegraf.py' to '/var/cache/salt/minion/extmods/modules/telegraf.py'
[INFO    ] Copying '/var/cache/salt/minion/files/base/_modules/zeekctl.py' to '/var/cache/salt/minion/extmods/modules/zeekctl.py'
[ERROR   ] Unable to connect pusher: Stream is closed
local:
    - modules.healthcheck
    - modules.needs_restarting
    - modules.so
    - modules.telegraf
    - modules.zeekctl
[INFO    ] Loading fresh modules for state activity
[INFO    ] Fetching file from saltenv 'base', ** done ** 'salt/minion.sls'
[INFO    ] Fetching file from saltenv 'base', ** done ** 'salt/map.jinja'
[INFO    ] Executing command 'yum' in directory '/root'
[INFO    ] Executing command rpm in directory '/root'
[INFO    ] Executing command 'yum' in directory '/root'
```

**Comment by launchdaemon** [comment]
Hi @m0duspwnens
I have been getting the `[ERROR   ] Unable to connect pusher: Stream is closed` error since updating to 2.3.100.

I have only noticed because I get an email each time `/usr/sbin/so-salt-minion-check -q` is run by cron but otherwise all services seem to be running fine.  The output of `so-status` is all `OK` and running `salt-call state.highstate` doesn't return anything unusal.

Also if I run the command `/usr/sbin/so-salt-minion-check` manually without `-q` the following is output:

```
running so-salt-minion-check
/opt/so/log/salt/healthcheck-state-apply last touched: Fri 25 Feb 09:56:12 UTC 2022 must be touched by Fri 25 Feb 10:56:12 UTC 2022 to avoid salt-minion restart
```

I'm not sure if it is related to the issue you are seeing during setup but am happy to provide further information or try and troubleshoot further if it will help.

**Comment by jertel** [comment]
Possibly related: 
- https://github.com/saltstack/salt/issues/61565
- https://github.com/saltstack/salt/issues/61521
